pandas/tests/groupby/test_only_unique.py

```diff
@@ -0,0 +1,52 @@
+import pytest
```

Comment:
These tests work when I force it to avoid trying the cython version. I don't know how to add a cython version yet.